docs/pages/components/icon.md

```diff
@@ -634,6 +634,40 @@ This example will load the same set of icons from the jsDelivr CDN instead of yo
 </script>
 ```
 
+### SVG Sprites
```

Comment:
Can this be moved above or below these two sections?

- Customizing the Default Library
- Customizing the System Library

It feels weird to break these up. If it's intended to be a subsection of the first section, maybe switch it to an `<h4>`.

Reply:
Moving below. Didnt even realize I just plopped it in the middle.

Reply:
Perhaps it does make sense to come after the `customizing the default library` and instead change the heading to this:

`### Customize the default library to use SVG sprites`